wordle-web/src/components/NavigationBar.vue

```diff
@@ -0,0 +1,147 @@
+      <v-list class="mt-2">
+        <v-list-item>
+          <v-btn
+            color="deep-orange-accent-3"
+            variant="outlined"
+            class="nav-btn"
+            @click="$router.push('/')"
+          >
+            Home
+          </v-btn>
+        </v-list-item>
+        <v-list-item>
+          <v-btn
+            color="deep-orange-accent-3"
+            variant="outlined"
+            class="nav-btn"
+            @click="$router.push('/wordle')"
+          >
+            Game
+          </v-btn>
+        </v-list-item>
+        <v-list-item>
+          <v-btn
+            color="deep-orange-accent-3"
+            variant="outlined"
+            class="nav-btn"
+            @click="$router.push('/about')"
+          >
+            About
+          </v-btn>
+        </v-list-item>
+        <!-- <v-list-item>
+          <v-btn
+            color="deep-orange-accent-3"
+            variant="outlined"
+            class="nav-btn"
+            @click="$router.push('/game')"
+          >
+            Working on Game
+          </v-btn>
+        </v-list-item> -->
+        <v-list-item>
+          <v-btn
+            color="deep-orange-accent-3"
+            size="medium"
+            class="ml-4 mt-1"
+            icon="mdi-close-circle-outline"
+            @click="drawer = !drawer"
+          ></v-btn>
+          <v-btn
+            color="light-green-accent-3"
+            size="medium"
+            class="pa-4 text-end"
+            icon="mdi-cog"
+            @click="dialog = true"
+          ></v-btn>
+          <!-- <v-dialog v-model="dialog" width="40%">
+            <v-card>
+              <v-card-title> Settings </v-card-title>
+              <v-select
+                label="Select Theme Color"
+                v-model="selectedTheme"
+                :items="myThemes"
+                @update:modelValue="setTheme()"
+                variant="solo"
+                hide-selected
+                class="mx-10"
+              ></v-select>
+              <v-card-actions>
+                <v-btn color="info" text @click="dialog = false">Close</v-btn>
+              </v-card-actions>
+            </v-card>
+          </v-dialog> -->
+        </v-list-item>
+      </v-list>
+    </v-navigation-drawer>
+    <v-btn
+      color="light-green-accent-3"
+      size="medium"
+      class="pa-4 text-end"
+      icon="mdi-cog"
+      @click="dialog = true"
+    ></v-btn>
+    <v-dialog v-model="dialog" width="40%">
+      <v-card>
+        <v-card-title> Settings </v-card-title>
+        <v-select
+          label="Select Theme Color"
+          v-model="selectedTheme"
+          :items="myThemes"
+          @update:modelValue="setTheme()"
+          variant="solo"
+          hide-selected
+          class="mx-10"
+        ></v-select>
+        <v-card-actions>
+          <v-btn color="info" text @click="dialog = false">Close</v-btn>
```

Comment:
This is typically put in the upper right, it's kind of weird having it here at the bottom